Packs/mnemonicMDR/pack_metadata.json

```diff
@@ -2,8 +2,7 @@
     "name": "mnemonic MDR",
     "description": "Rapidly detect, analyse and respond to security threats with mnemonic’s leading Managed Detection and Response (MDR) service.",
     "support": "partner",
-    "certification": "certified",
```

Comment:
Why did you remove it?

Reply:
Did not know this pack was certified. :) I'll re-add.